mempool/src/runtime.rs

```diff
@@ -27,8 +35,13 @@ impl MempoolRuntime {
     /// setup Mempool runtime
     pub fn bootstrap(
         config: &NodeConfig,
-        network_sender: MempoolNetworkSender,
-        network_events: MempoolNetworkEvents,
+        // (key, val) = (network ID, handle to network)
+        // See `NodeConfig::is_upstream_peer` for the definition of network ID
+        mempool_network_handles: HashMap<PeerId, MempoolNetworkHandle>,
```

Comment:
I don't think you actually need HashMap here or in inbound task
can we use vec of tuples instead?